database/database.py

```diff
@@ -0,0 +1,157 @@
+import sqlite3
+
+def connect_database():
+    conn = sqlite3.connect('database.db')
+    print("Database opened successfully")
+    return conn
+
+'''
+    Oxygen (Saturation): A Percentage
+        e.g 97% or 0.97
+    Pulse: Beats per minute
+        e.g. 80 beats per minute
+    Blood Pressure: systolic blood pressure / diastolic blood pressure
+        e.g. 120/80
+
+'''
+def create_tables(db):
+    print("Creating tables . . . ")
+    # Checks if table exists before creation
+    db.execute('create table if not exists Data (PatientID TEXT, Timestamp TEXT, Type TEXT, Value TEXT)')
+    db.execute('create table if not exists Alerts (PatientID TEXT, Timestamp TEXT, Type TEXT, ThresholdLow TEXT, ThresholdHigh TEXT, Value TEXT)')
+
+'''
+Description:
+    add_data_item adds one type (bp, o, p) item to the data database
+
+inputs:
+    patientID: The patient's identification code
+    timestamp: Time recorded
+    entrytype: Either Oxygen 'O', Pulse 'P', or Blood Pressure 'BP'
+    value:     The type's value to be stored
+'''
+def add_data_item(db, patientID, timestamp, entrytype, value):
+    if value is None:
+        return
+
+    db.execute("INSERT INTO Data (PatientID, Timestamp, Type, Value) \
+               VALUES (?,?,?,?)",(patientID, timestamp, entrytype, value) )
+    db.commit()
+    print("Data item added successfully")
+
+'''
+Description:
+    add_all_types_items adds up to all three items to the data database;
+    when calling, use parameterized function syntax
+
+inputs:
+    patientID:      The patient's identification code
+    timestamp:      Time recorded
+    oxygenValue:    Oxygen level, can be left out
+    pulseValue:     Pulse value, can be left out
+    bpValue:        Blood Pressure level, can be left out
+'''
+def add_all_types_items(db, patientID, timestamp, oxygenValue=None, pulseValue=None, bpValue=None):
+    add_data_item(db, patientID, timestamp, 'O', oxygenValue)
+    add_data_item(db, patientID, timestamp, 'P', pulseValue)
+    add_data_item(db, patientID, timestamp, 'BP', bpValue)
```

Comment:
You should change this to something like:
```
def add_all_types_items(oxygenValue=None, pulseValue=None, bpValue=None):
    timestamp = datetime.now()
    db = get_db_instance()
    patient_id = some_module.get_patient_id() # This should be from whatever module keeps track of the patient id
    add_data_item(db, patientID, timestamp, 'O', oxygenValue)
    add_data_item(db, patientID, timestamp, 'P', pulseValue)
    add_data_item(db, patientID, timestamp, 'BP', bpValue)
```

Alternatively, the patient ID could be generated by the data_collector and passed through by the scheduler, so we would have 4 inputs (patient_id, oxygen, pulse, bp).